x/staking/keeper/keeper.go

```diff
@@ -172,3 +179,161 @@ func (k Keeper) GetValidatorUpdates(ctx context.Context) ([]abci.ValidatorUpdate
 
 	return valUpdates.Updates, nil
 }
+
+// Copied from baseapp/abci_utils.go
```

Comment:
This function has been updated by the SDK team to address issue [17518](https://github.com/cosmos/cosmos-sdk/pull/17518) and the changes need to be updated here as well.

Reply:
cc: @VAIBHAVJINDAL3012